src/Setup/ImplementationOfInterfaceFinder.php

```diff
@@ -51,7 +51,7 @@ private function getAllClassNames() : \Iterator
             "|",
             array_map(
                 function ($v) { return "($v)"; },
```

Comment:
```suggestion
                // fix path-separators to respect windows' backspaces.
                function ($v) { return "(".str_replace($v, '/', '(/|\\\\)').")"; },
```

Should do the same (right?) and (for me) is more readable.

Reply:
@klees Yes, this does the same. I thought about a very similar solution but decided to used a separate array and merge these arrays :-).